themes/_default.yaml

```diff
@@ -114,37 +114,45 @@ badge:
     right-left: 8px
     top-bottom: 6px
 button:
-  icon:
-    color:
-      normal: "var(--tx-generic-color-primary-lighter)"
-      hover: "var(--tx-generic-color-active)"
-      active: "var(--tx-generic-color-primary-lighter)"
-  color:
-    bg-top: "var(--tx-generic-color-accent)"
-    bg-bottom: "var(--tx-generic-color-accent)"
-    text: "var(--tx-generic-color-blank)"
-    border: "var(--tx-generic-color-accent-dark)"
-    shadow: "transparent"
+  default:
+    normal:
+      background: "var(--tx-button-default-normal-background)"
```

Comment:
please adjust the changes as we discussed ;) 
also check other similar definitions :)

Reply:
yep! Thanks, Ivana))